awxkit/awxkit/utils/__init__.py

```diff
@@ -320,6 +321,22 @@ def update_payload(payload, fields, kwargs):
     return payload
 
 
+def to_str(obj):
+    if six.PY3:
```

Comment:
This is a little gross, and I suspect `six` has something to do this in a _cleaner_ way, but this is also code that I largely cherry-picked from the awxkit history *before* the py3 move.